src/System.Collections/tests/Generic/Dictionary/Dictionary.Generic.Tests.netcoreapp.cs

```diff
@@ -91,67 +92,107 @@ public void Dictionary_Generic_RemoveKey_DefaultKeyContainedInDictionary(int cou
         public void EnsureCapacity_NegativeCapacityRequested_Throws()
         {
             var dictionary = new Dictionary<string, string>();
-            Assert.Throws<ArgumentOutOfRangeException>(() => dictionary.EnsureCapacity(-1));
+            AssertExtensions.Throws<ArgumentOutOfRangeException>("capacity", () => dictionary.EnsureCapacity(-1));
+        }
+
+        [Fact]
+        [ActiveIssue("dotnet/corefx #24445")] 
```

Comment:
Marking this test as ActiveIssue until PR [coreclr 16076](https://github.com/dotnet/coreclr/pull/16076) gets merged.